src/app/server/Mdns.cpp

```diff
@@ -246,7 +241,18 @@ CHIP_ERROR Advertise(bool commissionableNode, CommissioningMode mode)
             advertiseParameters.SetPairingInstr(chip::Optional<const char *>::Value(pairingInst));
         }
     }
+    return CHIP_NO_ERROR;
+}
 
+/// Overloaded utility method for commissioner and commissionable advertisement
+/// This method is used for both commissioner discovery and commissionable node discovery since
+/// they share many fields.
+///   commissionableNode = true : advertise commissionable node
```

Comment:
true/false seems to be hard to follow: I read `Advertise(true)` as "yes, advertise" and `Advertise(false)` as 'no, do not advertise.

Please use an enum like `kCommisionableNode` and `kCommisioner` or similar (i.e. signature should look like `Advertise(AdvertiseNodeType nodeType, CommisioningMode mode)`